loan/request-quote.md

```diff
@@ -0,0 +1,45 @@
+---
+order: 9
+label: "Request Quote"
+---
+
+# Requesting a Quote with Votre
+
+Customize your personal loan experience by requesting a quote through Votre. This process is designed to be efficient and user-friendly, ensuring you receive competitive offers tailored to your needs.
+
+### How It Works
+
+=== 1. **Submit Your Request**
+
+Begin by filling out the necessary details for your loan request. This includes the loan amount, duration, and any specific terms you might have in mind.
+
+=== 2. **Supplier Network Engagement**
+
+Once submitted, Votre will publish your request to our curated network of suppliers. These suppliers will review your request and provide personalized offers in real-time.
```

Comment:
suppliers -> providers

"personalized" should be removed as implies sharing PII / KYC with providers